sourcegraph/sourcegraph
# notebooks: list page create dropdown button

## Screenshots

<img width="838" alt="Screenshot 2022-06-13 at 09 10 45" src="https://user-images.githubusercontent.com/6417322/173299153-cb2272f3-c322-4a6d-a95f-358b693314c3.png">
<img width="660" alt="Screenshot 2022-06-13 at 09 14 37" src="https://user-images.githubusercontent.com/6417322/173299546-4b9b8f34-a6e3-4960-95e9-ba8fa801f71f.png">

# Test plan



* Go to `/notebooks`
* Click on create notebook button -> it should create a blank notebook
* Click on the dropdown button, and click on the import markdown notebook item -> it should open a file dialog, select a markdown file -> it should create a new notebook populated with the markdown file content

## App preview:

- [Web](https://sg-web-rn-notebooks-list-page-dropdown.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-hjvotjxeuv.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
